src/integ_test_resources/flutter/amplify/github-cloudwatch-cdk/lib/createCanaryTestFailureAlarm.ts

```diff
@@ -0,0 +1,42 @@
+import { Duration } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { Alarm, ComparisonOperator, GraphWidget, Metric, TextWidget, TreatMissingData } from 'aws-cdk-lib/aws-cloudwatch';
+
+// Create the metric and alarms for tracking Github Canary failures.
+// These metrics are sent in Github action: amplify-flutter/amplify_canaries
+export function createCanaryTestFailureAlarm(scope: Construct) {
+  const createCanaryTestFailureMetric = new Metric({
+    metricName: "CanaryTestFailure",
+    namespace: "GithubCanaryApps",
```

Comment:
see [comment in other PR](https://github.com/aws-amplify/amplify-flutter/pull/3232/files#r1237760599), but I think there will be related details if this alarm is only for specific flutter channel, or maybe there should be separate alarms for different flutter channels